web/components/input/buttons/ProgressGroup/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ProgressGroup = ({ count, onClick, activeIndex, lastValidIndex }) => {
+const ProgressGroup = ({ count, onClick, activeIndex, lastValidIndex, classNames }) => {
```

Comment:
Yeah I see what you mean, but I also feel like it’s nice to have props that are structured instead of bunch of class names joined to a single string. If you feel strongly about it I’ll change it!

Reply:
I think it's best to change this to use a single-string `className` - you're right that structured props are almost always better, but in this case we are extending a native feature that has an existing convention, and it's probably less confusing to just follow go with it.